07/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="pl">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Zadanie 7</title>
+    <link rel="stylesheet" href="./styles/index.css" />
+</head>
+
+<body>
+    <header>
+        <section class="subheader">
+            <div class="logo">
+                <a></a>
+            </div>
+            <div class="menu">
+                <div class="menu-button">Menu</div>
+                <div class="menu-content">
+                    <a href="#o-nas">O nas</a>
+                    <a href="#kontakt">Kontakt</a>
+                    <a href="#wiecej">Więcej</a>
+                </div>
+            </div>
+        </section>
+    </header>
+
+    <div class="middle-content">
+        <section class="subsection">
+            <div class="container1">
+                <div class="object1">
+                    <a>obiekt1</a>
+                </div>
+                <div class="object2">
+                    <a>obiekt2</a>
+                </div>
+            </div>
```

Comment:
Sporo tych div-ów. Rozumiem, że to tylko przykład, ale już na takim przykładzie warto przyzwyczaić się do wykorzystywania bardziej semantycznych znaczników np. `article` czy `aside``